src/flashcart/ed64/ed64_pseudo_state.c

```
@@ -0,0 +1,48 @@
+#include <string.h>
+
+#include "ed64_pseudo_state.h"
+#include "menu/ini_parser.h"
+
+static char *ed64_pseudo_state_path = NULL;
+
+void ed64_pseudo_state_init (const char *path) {
+    if (ed64_pseudo_state_path) {
+        free(ed64_pseudo_state_path);
+    }
+    ed64_pseudo_state_path = strdup(path);
```

Clang (14.0.6)

[error] incompatible integer to pointer conversion assigning to 'char *' from 'int'

(clang-diagnostic-int-conversion)